internal/restapi/browse/handler.go

```diff
@@ -47,6 +51,10 @@ func (h *browseHttpService) createRouter(mountPath string) *mux.Router {
 	apiRouter.StrictSlash(true)
 	apiRouter.Handle("/modules/{organization_name}/{name}/{provider}", h.getModuleMetadataHandler()).Methods(http.MethodGet)
 	apiRouter.Handle("/modules", h.getModuleListHandler()).Methods(http.MethodGet)
+	apiRouter.Handle("/releases/{age}", h.getReleasesHandler()).Methods(http.MethodGet)
```

Comment:
Remove

Reply:
This is for the case when the age is specified and the next one is when the age is not specified. Not really sure why to remove this route as this is the main route used by the Front End?

Reply:
We don't need a route as you should be using a query parameter ie:
`/releases?age=48h` 
**not** this: `/releases/48h`

Reply:
Ah, I handled it similarly to `/modules/{organization_name}/{name}/{provider}` which is the route way and not query parameter. Will fix this